compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/input/pointer/SuspendingPointerInputFilter.kt

```diff
@@ -584,21 +584,21 @@ internal class SuspendingPointerInputModifierNodeImpl(
         if (lastEvent.changes.fastAll { !it.pressed }) {
             return // There aren't any pressed pointers, so we don't need to send any events.
         }
-        val newChanges = lastEvent.changes.fastMapNotNull { old ->
-            PointerInputChange(
-                id = old.id,
-                position = old.position,
-                uptimeMillis = old.uptimeMillis,
-                pressed = false,
-                pressure = old.pressure,
-                previousPosition = old.position,
-                previousUptimeMillis = old.uptimeMillis,
-                previousPressed = old.pressed,
-                isInitiallyConsumed = old.pressed
-            )
-        }
-
-        if (newChanges.isEmpty()) return
```

Comment:
Is this not needed anymore?

Reply:
it was not part of my changes - but this is the way it was in the androidx-main (and remained after resolving conflict)
so I can either - 

- return this line (it was introduced in 2021 in Ia813bd3da939315b7986e00a0a2dbb72df9c4aaa but never backported)
- live it in aosp

Reply:
Not sure I understand. If it was not part of your change, how did it make it into the commit?

Reply:
It was an artifact from the resolved conflict - luckily you've noticed it (no irony)
You are actually correct, in any way this should not be the part of this particular PR.